continuation.jl

```diff
@@ -1,14 +1,40 @@
 using PseudoArcLengthContinuation, Flux
-using PseudoArcLengthContinuation: ContResult,AbstractLinearSolver,EigenSolver
+using PseudoArcLengthContinuation: ContResult, DefaultLS, DefaultEig
 const Cont = PseudoArcLengthContinuation
 
+import Base: copyto!
+copyto!(x::TrackedArray, y::TrackedArray) where T = x = copy(y)
+
+import PseudoArcLengthContinuation: Newton, minus!
+minus!(x::TrackedArray, y::TrackedArray) = x = x .- y
+minus!(x::TrackedArray, y) = x = x .- y
+
+function fdTracked(F, x::AbstractVector; δ = 1e-9)
+	f = F(x)
+	N = length(x)
+	J = zeros(eltype(f), N, N)
+
+	Δx = zeros(N)
+	for i=1:N
+		Δx[i] += δ
+		J[:, i] .= (F(x.+Δx) .- F(x)) / δ
+		Δx[i] -= δ
+	end
+	return J
+end
+
```

Comment:
I think this should be in `finiteDifferences()`inside `src/Utils.jl`.
![Screenshot from 2019-10-13 23-32-52](https://user-images.githubusercontent.com/17977117/66723870-791ac700-ee16-11e9-9963-8ff287247666.png)

Reply:
But this finite differences function is slower than the one in `src/Utils.jl`. What's wrong with passing `fdTracked` it to `newton`?

Also, you should use `Zygote` instead of `fdTracked`...